src/containers/Header/index.js

```diff
@@ -11,11 +11,21 @@ import './header.scss';
 
 export function Header(props) {
 	const HeaderService = props.app.locateService("HeaderService");
-	const href = props.brand_image.href ?? "/";
 
 	const [headerProperties, setHeaderProperties] = useState(false);
 	const [windowDimensions, setWindowDimensions] = useState({width: window.innerWidth,});
 
+	const theme = useSelector(state => state.theme);
+	const title = useSelector(state => state.config?.title);
+
+	const BrandingService = props.app.Services.BrandingService;
+
+	let brandImage;
+	if (BrandingService && theme) {
+		brandImage = BrandingService.getLogo(theme);
+	}
+	const href = brandImage?.href ?? "/";
```

Comment:
@petrKavulok This condition is redundant, since branding service always return brand image href

```
href: imgObject.href ?? '/'
```